topo_processor/cli/main.py

```diff
@@ -6,7 +6,7 @@
 import click
 from linz_logger import get_log
 
-from topo_processor.metadata import DataType, create_collection
+from topo_processor.factory import DataType, create_collection
```

Comment:
factory is super generic name when its creating metadata why the change?

Reply:
because it also calls compress data / tiffs in create_collection

Reply:
I have renamed it to stac_object because they are mainly for creating collections and items. Thoughts? @blacha

Reply:
have renamed it to stac